### ISSUE TITLE
Compare two FixedTree indices for equivalence

### ISSUE THREAD

**Issue by eblume** [eblume/Tiger-Lily#3]
I need to look up the docs to make sure I have this right, but this probably means writing an **equals** function. This should compare the tree structures and ascertain that identical alignments are present.

This is NOT a mandatory milestone feature, but it is still a good idea to have it.

**Comment by eblume** [comment]
This is being put on hold because it looks like we need to rewrite the class mostly in C - performance is far too low.

**Comment by eblume** [comment]
Blocked by #18
